Source/ToolCore/Build/AndroidProjectGenerator.cpp

```diff
@@ -135,10 +140,35 @@ bool AndroidProjectGenerator::GenerateLocalProperties()
     File file(context_, buildPath_ + "/local.properties", FILE_WRITE);
 
     if (!file.IsOpen())
+    {
+        errorText_ = "Project generator unable to open file " + buildPath_ + "/local.properties ";
         return false;
-
+    }
+    
     file.Write(props.CString(), props.Length());
 
+
+    if ( prefs->GetReleaseCheck() > 0 ) // if release flag is set ...
+    {
+        FileSystem* fileSystem = GetSubsystem<FileSystem>();
+        String Reldir = prefs->GetReleasePath();
+        if (!fileSystem->DirExists(Reldir))
+        {
+            errorText_ = "Invalid Release Path, select the path in Build Settings.";
+            return false;
+        }
+        
+        String antname = Reldir + "/ant.properties";
+        if ( !fileSystem->FileExists ( antname) ) 
+        {
+            errorText_ = "The file ant.properties not found in " + Reldir + ", unable to generate Release APK.";
+            return false;
+        }
+
+        Poco::File antp ( antname.CString() ); // lookup ant.properties 
+        antp.copyTo( buildPath_.CString());  // copy offending file
```

Comment:
Another note, it is good to use the build system directory creation, file copies, etc and not Poco as the build system ones check for and propagate errors: https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/ToolCore/Build/BuildWindows.cpp#L132